credoai/integration.py

```diff
@@ -71,21 +67,20 @@ class Metric(Record):
     def __init__(self,
```

Comment:
@aproxacs , changes so far. Will add the "metric key" when you tell me what that is